### DISCUSSION TITLE
Full SCENIC Pipeline for chicken

### DISCUSSION THREAD

**Discussion by austinkunch** [aertslab/SCENIC#412] [* General SCENIC questions]
Hi, I see there are Motif to TF annotations for **Chicken** in cistargetdb, but for Databases ranking the whole genome of the species, there is only data for human, mouse, fly. Does anyone know where I can find Feather files for ranking the whole genome of chicken to be used with the Chicken Motif to TF annotations (in pySCENIC)?

Also, I have batch effects in my data. Should I normalize and correct for batch effects before running data through the scenic pipeline, or pass the raw counts through the pipeline without normalization and batch effect correction?

Thank you!

**Comment by DiracZhu1998** [comment]
> Hi, I see there are Motif to TF annotations for **Chicken** in cistargetdb, but for Databases ranking the whole genome of the species, there is only data for human, mouse, fly. Does anyone know where I can find Feather files for ranking the whole genome of chicken to be used with the Chicken Motif to TF annotations (in pySCENIC)?
> 
> Also, I have batch effects in my data. Should I normalize and correct for batch effects before running data through the scenic pipeline, or pass the raw counts through the pipeline without normalization and batch effect correction?
> 
> Thank you!

Hi Austin, I was wondering have you managed to solve that ? Thanks!
